index.src.html

```diff
@@ -1885,30 +1889,41 @@
 ---------------------------------------------
 <xmp class='idl'>
 enum HardwareAcceleration {
```

Comment:
Given we might introduce in the future other values (prefer-compression-efficiency on encoder side maybe), I would tend to rename the enum and the config field name itself to something like codecPreference.